shape_similarity.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+
+import time
+import argparse
+import openeye
+from openeye.oechem import *
+from openeye.oeomega import *
+from openeye.oeshape import *
```

Comment:
This might be a stylistic choice, but I am not a fan of * imports :\ it makes it very tricky to figure out exactly what is used here, and makes it harder down the road to maintain with library changes.